figures/scripts/fig2-oncoprint-landscape.R

```diff
@@ -398,14 +407,14 @@ for (type_iter in seq_along(data_input_list)) {
         annotationFontSize = 1.25,
         SampleNamefontSize = 1,
         annotationColor = list(cancer_group = supp_colors,
-                               germline_sex_estimate = c("Male" = "#2166ac",
-                                                         "Female" = "#b2182b")),
+                               germline_sex_estimate = germline_sex_estimate_colors),
         fontSize = 1,
         colors = oncoprint_palette,
         bgCol = "#F5F5F5",
         drawRowBar = FALSE,
-        titleText = histology,
-        titleFontSize = 1.3,
+        showTitle = FALSE, # a legible font size is not possible to fit into margins
+        #titleText = histology,
+        #titleFontSize = 1.3,
```

Comment:
Leave it in anyway. It will be less error prone to increase the font size in a vector image editor over adding it in the compiled figure!

Reply:
Titles are now restored